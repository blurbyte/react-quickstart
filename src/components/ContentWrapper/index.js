import styled from 'styled-components';

const ContentWrapper = styled.article`
  margin: 0 auto;
  padding: 10rem 6.4rem;
  max-width: 1440px;
`;

export default ContentWrapper;
