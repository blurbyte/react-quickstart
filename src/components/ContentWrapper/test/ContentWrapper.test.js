import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ContentWrapper from '../index';

describe('<ContentWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <ContentWrapper />
    );

    expect(component.toJSON()).toMatchStyledComponentsSnapshot();
  });
});
