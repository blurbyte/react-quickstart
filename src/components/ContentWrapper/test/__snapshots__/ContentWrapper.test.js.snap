// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentWrapper /> should render correctly 1`] = `
.c0 {
  margin: 0 auto;
  padding: 10rem 6.4rem;
  max-width: 1440px;
}

<article
  className="c0"
/>
`;
